notebooks/official/tabular_workflows/bqml_arima_plus.ipynb

```diff
@@ -1121,10 +1089,7 @@
         "To clean up all Google Cloud resources used in this project, you can [delete the Google Cloud\n",
         "project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#shutting_down_projects) you used for the tutorial.\n",
         "\n",
-        "Otherwise, you can delete the individual resources you created in this tutorial:\n",
-        "\n",
-        "- Cloud Storage Bucket\n",
-        "- BigQuery tables"
+        "Otherwise, you can delete the individual resources you created in this tutorial"
```

Comment:
add "that":  "Otherwise, you can delete the individual resources that you created in this tutorial"